frontend/src/metabase/static-viz/containers/StaticChart/types.ts

```diff
@@ -0,0 +1,8 @@
+import { STATIC_CHART_TYPES } from "./constants";
+
+type StaticChartType = typeof STATIC_CHART_TYPES[number];
+
+export interface IStaticChartProps {
+  type: StaticChartType;
+  options: any;
```

Comment:
`unknown`?

Reply:
`unknown` is throwing a TypeScript error regarding passed props being incompatible...
IMHO, if `StaticChart` is allowing to render all chart components, all the components `props` should use the same interface.